internal/eval_checks/diagnostics.go

```diff
@@ -3,7 +3,7 @@
 // Copyright (c) 2023 HashiCorp, Inc.
 // SPDX-License-Identifier: MPL-2.0
 
-package tofu
+package evalchecks
```

Comment:
can we make the package name the same as the folder name?

Reply:
oh yeah. i just lazily threw the package here until we figured out where it should go.. then never did..